cmd/token/create.go

```diff
@@ -61,7 +62,7 @@ k0s token create --role worker --expiry 10m  //sets expiration time to 10 minute
 			}, func(err error) bool {
 				return waitCreate
 			}, func() error {
-				bootstrapConfig, err = token.CreateKubeletBootstrapConfig(clusterConfig, c.K0sVars, createTokenRole, expiry)
+				bootstrapConfig, err = token.CreateKubeletBootstrapConfig(context.TODO(), clusterConfig, c.K0sVars, createTokenRole, expiry)
```

Comment:
here and everywhere under the cmd/ package you should instead of context.TODO use context derived from the cobra.Command